### PR TITLE
specific difficulty tests to develop

### PR DESCRIPTION
**Pull request by winsvega**

### CODE REVIEW

File: test/fuzzTesting/createRandomTest.cpp [hunk 1]
```diff
@@ -197,8 +193,8 @@ void fillRandomTest(std::function<void(json_spirit::mValue&, bool)> _doTests, st
 	json_spirit::mValue v;
 	try
 	{
-		std::string newTest = _testString;
-		parseTestWithTypes(newTest);
+		std::string newTest = _testString;
```
Comment (CJentzsch) [inline test/fuzzTesting/createRandomTest.cpp]: please remove whitespaces/tabs at the end

File: test/fuzzTesting/fuzzHelper.h [hunk 2]
```diff
@@ -78,6 +78,9 @@ class RandomCode
 	/// Generate random vm code
 	static std::string generate(int _maxOpNumber = 1, RandomCodeOptions _options = RandomCodeOptions());
 
+	/// Replace keywords in given string with values
+	static void parseTestWithTypes(std::string& test, std::map<std::string, std::string> const& _varMap = {});
```
Comment (CJentzsch) [inline test/fuzzTesting/fuzzHelper.h]: `_test` instead of `test`

**Comment by CJentzsch** [comment]
Those commits are already merged in `hot_gav` so lets fix those minor style issues later to avoid merge conflicts.

**Comment by winsvega** [comment]
https://github.com/ethereum/libethereum/pull/111

**Comment by CJentzsch** [comment]
tests failing.

**Comment by chriseth** [comment]
Can you rebase this to see whether tests are green now?

**Comment by winsvega** [comment]
it wont be green anyways. cpp client has a different difficulty formula implemented.
though it is not a critical issue. just in some cases of realization on earliest blocks

**Pull request merged**